### DISCUSSION TITLE
<Images are broken>

### DISCUSSION THREAD

**Discussion by gopi-vk** [community/community#56933] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am facing this issues about a month while uploading the images in Git. It is uploading and able to preview but when it is coming to [docs.int.company_name.com](https://docs.int.company_name.com/-/Harvest-Issues-Standardization/preview/5142009177/GWS%20Understanding/General_Flow/) page image is not visible. Requesting to provide me solution asap.

**Comment by Venkatesh-KCET** [comment]
Hey @gopi-vk, 
could you please provide more information about it? 
Additionally, the link you shared doesn't seem to be working. Could you please check and provide an alternative link if possible? 

Thanks!
